src/components/VideoHero/VideoHero.tsx

```diff
@@ -111,19 +112,15 @@ export const VideoHero: React.FC = () => {
             appear
           >
             <ButtonsContainer>
```

Comment:
We need to handle this appearing, right now there's a jump in the title. Before this change, the space for buttons was preserved, so when they appeared nothing changed places